src/mame/taito/taito_f3_v.cpp

```diff
@@ -2265,6 +2259,8 @@ void taito_f3_state::scanline_draw(bitmap_rgb32 &bitmap, const rectangle &clipre
 		}
 
 		/* set scanline priority */
+		std::array<u8, 5> layer_tmp;
+		int count_skip_layer = 0;
```

Comment:
Does changing `layer_tmp` to `std::array` actually help?  You can still use `std::sort` on a C array (using `std::begin` and `std::end` to get the begin/end iterators).

Reply:
Ah, didn't know about std::begin & end! Changed back to C array and used those.